app/interactors/articles/create.rb

```diff
@@ -0,0 +1,8 @@
+module Articles
+  class Create
+    include Interactor::Organizer
+
+    organize Prepare,
+             Save
```

Comment:
Давай тут на всякий случай явно модули пропишем чтобы не было конфликтов в имени файлов

```suggestion
    organize Articles::Create::PrepareParams,
             Articles::Save
```